source/2020-04-10-the-ember-times-issue-143.md

```diff
@@ -57,14 +58,15 @@ We encourage you to check out the Ember CLI documentation to understand your too
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [Introducing qunit-wait-for ⏰](https://twitter.com/alexlafroscia/status/1246102930475962368)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+There's an awesome new test helper by [Alex LaFroscia @alexlafroscia](https://github.com/alexlafroscia) that rethinks the approach on how we wait for asyncronous behavior to resolve in our tests! 🎉
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+Typically in Ember tests you'd use one of the several available test helpers that wait until a promise resolves before making some kind of assertion against the state of your application. However, using wait test helpers in this way can add complexity to your tests and couple ⛓ your tests to your implementation code. 
```

Comment:
I think we can provide a reader more context by providing a link to `@ember/test-helpers` doc:
```suggestion
Typically in Ember tests you'd use one of the several available test helpers that [wait until a promise resolves](https://github.com/emberjs/ember-test-helpers/blob/master/API.md#waituntil) before making some kind of assertion against the state of your application. However, using wait test helpers in this way can add complexity to your tests and couple ⛓ your tests to your implementation code. 
```